isis/src/base/objs/UserInterface/UserInterface.cpp

```diff
@@ -405,22 +405,22 @@ namespace Isis {
    */
   void UserInterface::loadCommandLine(QVector<QString> &args, bool ignoreAppName) {
     char **c_args;
-    
+
     if (ignoreAppName) {
       args.prepend("someapp");
     }
 
     c_args = (char**)malloc(sizeof(char*)*args.size());
-    
-    for (size_t i = 0; i < args.size(); i++) {
-      c_args[i] = (char*)malloc(sizeof(char)*args[i].size()+1);
+
+    for (int i = 0; i < args.size(); i++) {
+      c_args[i] = (char*)malloc(sizeof(char)*args[i].size());
```

Comment:
The +1 one needs to put back, this is why the tests are failing. The +1 makes sure the null terminator is copied over when creating the c-string. 

see: https://www.tutorialspoint.com/what-is-a-null-terminated-string-in-c-cplusplus